security-openid-connect-multi-tenancy quickstart fails with Quarus master

security-openid-connect-multi-tenancy quickstart fails with Quarus master

@sberyozkin fyi

```
2020-04-21 08:28:51,891 INFO  [io.quarkus] (main) Installed features: [cdi, oidc, resteasy, resteasy-jsonb, security]
2020-04-21 08:28:55,777 INFO  [com.gar.htm.WebClient] (main) statusCode=[401] contentType=[]
2020-04-21 08:28:55,784 INFO  [com.gar.htm.WebClient] (main)
[ERROR] Tests run: 3, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 47.483 s <<< FAILURE! - in org.acme.quickstart.oidc.CodeFlowTest
[ERROR] testReAuthenticateWhenSwitchingTenants  Time elapsed: 3.912 s  <<< ERROR!
com.gargoylesoftware.htmlunit.FailingHttpStatusCodeException: 401 Unauthorized for http://localhost:8081/unknown
	at org.acme.quickstart.oidc.CodeFlowTest.testReAuthenticateWhenSwitchingTenants(CodeFlowTest.java:74)

2020-04-21 08:28:58,483 INFO  [io.quarkus] (main) Quarkus stopped in 0.028s
[INFO]
[INFO] Results:
[INFO]
[ERROR] Errors:
[ERROR]   CodeFlowTest.testReAuthenticateWhenSwitchingTenants:74 » FailingHttpStatusCode
[INFO]
[ERROR] Tests run: 3, Failures: 0, Errors: 1, Skipped: 0
```

--- a/security-openid-connect-multi-tenancy/src/test/java/org/acme/quickstart/oidc/CodeFlowTest.java
+++ b/security-openid-connect-multi-tenancy/src/test/java/org/acme/quickstart/oidc/CodeFlowTest.java
@@ -5,7 +5,6 @@ import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import java.io.IOException;
 
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 
@@ -57,7 +56,6 @@ public class CodeFlowTest {
     }
 
     @Test
-    @Disabled
     public void testReAuthenticateWhenSwitchingTenants() throws IOException {
         try (final WebClient webClient = createWebClient()) {
             HtmlPage page = webClient.getPage("http://localhost:8081/tenant-a");

